### DISCUSSION TITLE
How to use other domain than .com in extends

### DISCUSSION THREAD

**Discussion by Bean900** [renovatebot/renovate#9570] [Request Help] [answered]
Hey :)

We want to use a central renovate.json for our project. Now we have the problem that every domain is resolved as a .com domain (Ref.: https://docs.renovatebot.com/config-presets/#preset-hosting).
In our case, the domain looks something like this:
`https://git.abc.xyz.de/`

My first idea would be to write it like this:
 
```
"extends": [
    "git.abc.xyz>..."
  ]
```

Then renovate would recognize it as a .com domaine, would it?
`https://git.abc.xyz.com/`

How can we use other domains than .com?

**Comment by viceice** [comment, marked as answer]
You can't use arbitrary domains here. Use `local` shared preset or a known hosted platform like github. The docs you mentioned should cover this
